Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue

```diff
@@ -26,20 +26,13 @@ export default class FilterComponent extends Vue {
     config!: WebClientConfiguration;
     @Getter("isOpen", { namespace: "sidebar" }) isSidebarOpen!: boolean;
 
-    @Prop() private medicationCount!: number;
-    @Prop() private immunizationCount!: number;
-    @Prop() private encounterCount!: number;
-    @Prop() private laboratoryCount!: number;
-    @Prop() private noteCount!: number;
-    @Prop() private isListView!: boolean;
     @Prop() private filter!: TimelineFilter;
 
     private logger!: ILogger;
     private eventBus = EventBus;
     private isVisible = false;
     private windowWidth = 0;
-    private steps = [25, 50, 100, 500];
```

Comment:
steps are still necessary since we are not removing pagination just yet

Reply:
@FuriousLlama @ninosamson I believe we want to remove the steps right? 
![image](https://user-images.githubusercontent.com/48332333/106499601-a2c98880-6475-11eb-98c1-ac551d73298a.png)